addons/mail/static/src/new/thread/message_in_reply_to.js

```diff
@@ -0,0 +1,64 @@
+/** @odoo-module */
+
+import { Component, useEffect, useState } from "@odoo/owl";
+
+import { useService } from "@web/core/utils/hooks";
+
+import { useMessaging } from "@mail/new/messaging_hook";
+
+export class MessageInReplyTo extends Component {
+    setup() {
+        this.messaging = useMessaging();
+        this.user = useService("user");
+        this.state = useState({
+            isInChatWindow: this.messaging.chatWindows.some(
+                ({ threadId }) => threadId === this.props.message.resId
+            ),
+        });
+        useEffect(
+            (chatWindows) => {
+                this.state.isInChatWindow = chatWindows.some(
+                    ({ threadId }) => threadId === this.props.message.resId
+                );
+            },
+            () => [this.messaging.chatWindows]
+        );
```

Comment:
Not sure about this one. A getter would lead to one computation per call. Even if there was only was call of the said getter, it would be called on every re-render. Traversing an array is not cheap, I think it's more efficient to keep it this way. What do you think ?

Reply:
No perceived performance issue, and right now it's more code maintenance by adding `useEffect`.
Note that you could make it work even without `useState` and just an attribute on the component.
But I really suggest doing even less code maintenance with just a getter.

Reply:
Done :+1: